addons/weave-net/weave-net.yaml

```diff
@@ -117,14 +129,15 @@ items:
                       apiVersion: v1
                       fieldPath: spec.nodeName
               image: 'docker.io/weaveworks/weave-kube:2.5.1'
+              imagePullPolicy: Always
               readinessProbe:
                 httpGet:
                   host: 127.0.0.1
                   path: /status
                   port: 6784
               resources:
                 requests:
-                  cpu: 10m
+                  cpu: 50m
```

Comment:
I think it would be worth mentioning this change in the commit description - it makes it more likely that Weave Net will have enough CPU to run, and less likely that the cluster will start on a really small machine.

Reply:
fixed. added your wording to describe that in a commit message.